From 18b458e470b42bce498d806c5bd0f576100c289d Mon Sep 17 00:00:00 2001
From: Peng Ma <peng.ma@nxp.com>
Date: Tue, 4 Dec 2018 15:59:16 +0800
Subject: [PATCH 722/767] ahci: qoriq: fixed the compile error

The powerpc arch will compile failed after applied the patch:
 ahci: qoriq: fixed lx2160 sata problem with errata A-379364
This patch fixed it.

Signed-off-by: Peng Ma <peng.ma@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/ata/ahci.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/ata/ahci.h b/drivers/ata/ahci.h
index 58867e014948..e5e99c3da70d 100644
--- a/drivers/ata/ahci.h
+++ b/drivers/ata/ahci.h
@@ -450,9 +450,9 @@ static inline int ahci_nr_ports(u32 cap)
 }
 
 #ifdef CONFIG_AHCI_QORIQ
-void fsl_sata_errata_379364(struct ata_link *link);
+extern void fsl_sata_errata_379364(struct ata_link *link);
 #else
-void fsl_sata_errata_379364(struct ata_link *link)
+static void fsl_sata_errata_379364(struct ata_link *link)
 {}
 #endif
 
-- 
2.17.0

